packages/sdk/src/utils/verifyClient.ts

```diff
@@ -3,61 +3,83 @@
  * Licensed under the MIT License.
  */
 
+import semver from 'semver';
 import * as Constants from '../constants';
+import { log } from '../log';
 
 /*
- * Supported protocol version
- * **WARNING**: Incrementing this value will break older clients. Strive to be backward compatible. Change this value
- * only after discussing with the team and considering every alternative.
+ * Current SDK Version - Read from package.json.
+ */
+// tslint:disable-next-line:no-var-requires variable-name
+const CurrentSDKVersion = semver.coerce(require('../../package.json').version);
+
+/*
+ * Minimum Supported Client Library version
+ * As part of the manual SDK release procedures, this is reset to match CurrentSDKVersion Major.Minor if new functions
+ * have been added that don't work on older clients (i.e. pretty much every release). Since host apps are required to
+ * update client libraries regularly, this one is not a big deal to update.
  */
 // tslint:disable-next-line:variable-name
-const CurrentProtocolVersion = 1;
+const MinimumSupportedClientVersion = semver.coerce('0.5');
 

```

Comment:
Does this mean that client versions are expected to track their SDK versions?

Reply:
The client needs to track the minimum SDK version it supports, but it shouldn't change often. We must make efforts to remain as backward compatible as possible. Incrementing this version in the client is a Big Deal, and requires team discussion and signoff.